packages/editor/src/components/post-slug/index.js

```diff
@@ -41,25 +42,19 @@ export class PostSlug extends Component {
 	}
 
 	render() {
-		const { instanceId } = this.props;
 		const { editedSlug } = this.state;
```

Comment:
The slug change from URL form isn't reflected in the slug text field without page refresh.

Reply:
Looks like this bug exists on `trunk` too so I'm just going to plow ahead.

For context I think this component was added back when we used the post editor for templates and template parts but was never removed and now we're stuck maintaining it. In theory a plugin can add `add_post_type_support( 'post', 'slug' )` but I do not see any evidence that this is happening.